mod_form.php

```diff
@@ -386,11 +386,15 @@ private function bigbluebuttonbn_mform_add_block_room_room(&$mform, $cfg) {
      * @param array $cfg
      * @return void
      */
-    private function bigbluebuttonbn_mform_add_block_guestlink(&$mform, $cfg) {
+    private function bigbluebuttonbn_mform_add_block_guestlink(&$mform, $cfg, $context) {
         if ($cfg['participant_guestlink']) {
             $mform->addElement('header', 'guestlink', get_string('mod_form_block_guestlink', 'bigbluebuttonbn'));
+            $checkboxattributes = [];
+            if (!(has_capability('mod/bigbluebuttonbn:guestlink_configure_access', $context))) {
+                $checkboxattributes['disabled'] = true;
```

Comment:
We need some serverside validation around this, otherwise someone could just hack the DOM and submit

Reply:
Validation added @ mod/bigbluebuttonbn/mod_form.php:183